src/Services/StoreFileService.php

```diff
@@ -13,7 +14,7 @@ class StoreFileService implements StoreFile
 
     public function storeFile(string $uploadFolder, UploadedFile $file): string
     {
-        $this->path = $file->store($uploadFolder, "public");
+        $this->path = Constants::STORAGE_PATH . $file->store($uploadFolder, "public");
```

Comment:
You should use Laravel helper `storage_path()`. It it will be tricky with creating app from our package, but we will return to this later.

Reply:
`storage_path()` is returning `/application/storage` path and this path is not working with views. Should I stay with that solution for this moment?

Reply:
And what's storage location then?